blockchain/proxy.go

```diff
@@ -559,6 +559,58 @@ func (ep *proxy) GetValidatorsInfoByEpoch(ctx context.Context, epoch uint32) ([]
 	return response.Data.ValidatorsInfo, nil
 }
 
+// GetESDTTokenData returns the address' fungible token data
+func (ep *proxy) GetESDTTokenData(ctx context.Context, address erdgoCore.AddressHandler, tokenIdentifier string) (*data.ESDTFungibleTokenData, error) {
```

Comment:
can be only a TODO for now, but we also support block coordinates in various endpoints (for example: "?blockNonce=373737"). We might add support for this as well in the sdk(s)

Reply:
done & added TODO for the other routes. Also, added TODO to move the function in the core repo and remove it from SDK & proxy.